shell/platform/linux/fl_engine_private.h

```diff
@@ -175,6 +175,41 @@ GBytes* fl_engine_send_platform_message_finish(FlEngine* engine,
                                                GAsyncResult* result,
                                                GError** error);
 
+/**
+ * fl_engine_mark_texture_frame_available:
+ * @engine: an #FlEngine.
+ * @texture_id: an int64_t.
```

Comment:
This should document what `texture_id` represents, e.g. 'the identifier of the texture whose frame has been updated'. (Same for cases below).